packages/chart/src/components/LineChart/components/LineChart.Circle.tsx

```diff
@@ -199,8 +205,9 @@ const LineChartCircle = (props: LineChartCircleProps) => {
     }
 
     if (drawIsolatedPoints(dataIndex, seriesKey)) {
```

Comment:
on 200 add: 

const _dataIndex = pointData ? data.findIndex(item => item[config.xAxis.dataKey] === pointData[config.xAxis.dataKey]) : dataIndex

and on line 201 reference _dataIndex